assets/src/console/pages/gateway-accounts/CardTypesPage.tsx

```diff
@@ -0,0 +1,122 @@
+import * as React from "react";
+import { Helmet } from "react-helmet";
+import createDecorator from "final-form-focus";
+
+import {
+  PageTitle,
+  P,
+  Checkbox,
+  Button,
+  Form,
+  Hr,
+  Strong,
+  Loader,
+  ErrorAlert
+} from "@pay/web";
+
+import {
+  GatewayAccountFragment,
+  useUpdateGatewayAccountCardTypesMutation,
+  CardTypeBrand,
+  CardTypeType
+} from "../../__generated__/graphql";
+import {
+  useCardTypesQuery,
+  CardTypesQuery
+} from "../../../platform-admin/__generated__/graphql";
+import { CardType } from "../../../auth/__generated__/graphql";
+
+interface FormValues {
+  cardTypes: Array<string>;
+}
+
+const decorators = [createDecorator<FormValues>()];
+
+interface Props {
+  gatewayAccount: GatewayAccountFragment;
+}
+
+const getCardTypeBrand = (cardType: CardType) => {
+  if (cardType.type && cardType.label) {
+    let label =
+      cardType.brand === CardTypeBrand.Visa ||
+      cardType.brand === CardTypeBrand.MasterCard
+        ? cardType.label + " " + cardType.type.toLowerCase()
+        : cardType.label;
+    return label;
+  } else {
+    return "";
+  }
+};
+
+const getCardTypesList = (data: CardTypesQuery, type: string) =>
+  data &&
+  data.cardTypes
+    .filter(cardType => cardType.type === type)
+    .map(cardType => {
```

Comment:
can just return the value inline instead of using `return { <>...</> }`—less code, easier to follow.

also can you check if the react component `key={cardType.id` can go on the fragment? i think it is meant to because it should go on the thing returned from mapping the array. i don't think it does anything where it is atm.